settings_sample.json

```diff
@@ -50,6 +50,15 @@
         }
     },
 
+    "//2.2": "Send an e-mail with a verification token to new user",
+    "sendVerificationEmail": false,
+
+    "//2.3": "Enable form to resend the verification token to users e-mail",
+    "showResendVerificationEmailLink": false,
+
+    "//2.4": "Enable form to perform user password reset",
+    "showForgotPasswordLink": false,
+
```

Comment:
All these three options work only with EMail enabled. So, please push these three settings one level down. Inside the existing `email { ... }` section.